src/cls/_ZPM/Main.cls

```diff
@@ -2120,14 +2009,54 @@ ClassMethod Import(ByRef pCommandInfo)
 					// If a file has previously been exported to the Perforce workspace but not actually marked for add, GetStatus will report that the file is
 					// in source control. This case is handled later after revert is attempted and fails because the file is not known to Perforce.
 					Kill ^Sources(0,"FileTimeStamp",tLoadedItem)
-					If $$$ISOK(##class(%Studio.SourceControl.Interface).GetStatus(tLoadedItem,.tIsInSourceControl)) {
-            If tIsInSourceControl {
-              Set tSC = $$$ADDSC(tSC,%SourceControl.OnAfterSave(tLoadedItem))
-            } Else {
-              Set tSC = $$$ADDSC(tSC,%SourceControl.AddToSourceControl(tLoadedItem))
-            }
-          }
```

Comment:
What are these changes? Even if they are well warranted/required, do not belong in this PR